Sources/ProjectAutomation/Settings.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+// A group of settings configurations.
+
+public typealias BuildConfigurationDictionary = [ProjectAutomation.BuildConfiguration: ProjectAutomation.Configuration]
+
+public struct Settings: Equatable, Codable {
+    /// A dictionary with build settings that are inherited from all the configurations.
```

Comment:
This is not a dictionary, right?

Reply:
yes, thy will remove this comment. was a copy paste